src/liquidationBot/services/traders.ts

```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
+import { config } from "@config";
+import type { Trader } from "@liquidationBot/types";
+
+const { tradesUrl } = config;
+
+export const getOpen = async (): Promise<Trader[]> => {
+  const { data } = await axios.get<{ trades: { trader: Trader }[] }>(tradesUrl);
+  return data.trades.map(({ trader }) => trader);
```

Comment:
Shouldn't this deduplicate the traders list? This seems to contain a lot of duplicates if it maps trade -> trader 1:1.